www/index.html

```diff
@@ -64,7 +64,7 @@
                     navigator.app.exitApp();
                 }
             }
-            if (window.location.hash === '#/') {
+            if (window.location.hash.startsWith('#/?_')) {
```

Comment:
it does not works, remember I mentioned `pathname` gives the whole path in mobile

any alternative?